docs/key_management.md

```diff
@@ -0,0 +1,175 @@
+## Uport android signer
+
+This library is used to create and manage keys for uport account.
```

Comment:
```suggestion
The `signer` library is used to create and manage keys for uport accounts.
It can be used independently but it is ultimately meant to be used as part of the android SDK, wrapped behind easier to use abstractions (like the `Signer` interface, or an `Account` object).
```